Dockerfile

```
@@ -48,7 +48,24 @@ RUN mkdir /opt/litecoin && cd /opt/litecoin \
 FROM debian:buster-slim as builder
 
 ENV LIGHTNINGD_VERSION=master
-RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates autoconf automake build-essential git libtool python3 python3-pip python3-setuptools python3-mako wget gnupg dirmngr git gettext
+RUN apt-get update -qq && \
+    apt-get install -qq -y --no-install-recommends \
+        autoconf \
+        automake \
+        build-essential \
+        ca-certificates \
+        dirmngr \
+        gettext \
+        git \
+        gnupg \
+        libtool \
+        python3 \
+        python3-mako \
+        python3-pip \
+        python3-setuptools \
+        wget
+
+RUN pip3 install -U setuptools mrkd mako
```

```suggestion
RUN pip3 install -U setuptools mrkd mako mistune==0.8.4
```

Little worried, it here we need to set also the `mistune==0.8.4`, but I didn't check if the dev of `mrkd` pushed the new version on `pypi`

---

Just use requirements.lock which has a known good combination of dependencies resolved to their respective versions. Having to replace all these tiny version pins whenever we upgrade is tedious.

---

ops! Sorry if I review when you are working on it, concurrency is always bad!

In addition, I like the idea to use `requirements.lock` everywhere, but in this case, I did mention it because I thought that maybe with the release we don't need some dependencies.

But I was wrong :)